### PR TITLE
Bc logo header footer

### PR DESCRIPTION
**Pull request by plondon**

### CODE REVIEW

File: app/partials/bc-logo-header.pug [hunk 1]
```diff
@@ -0,0 +1,5 @@
+nav.navbar.navbar-default.navbar-inverse.pos-abs.bc-header.bc-logo-header.width-100.width-100-tablet.width-100-mobile(style="border-radius:0px; border: 0px;")
```
Comment (jtormey) [inline app/partials/bc-logo-header.pug]: Is this inline style necessary? For me it creates a CSP error.

File: app/partials/bc-logo-footer.pug [hunk 2]
```diff
@@ -0,0 +1,16 @@
+.flex-row.flex-center.flex-column-mobile.flex-between.bc-logo-footer.width-100.width-100-tablet
+  .container
+    .flex-row.flex-center.flex-between.flex-column-mobile
+      .flex-row.flex-center.flex-column-mobile.em-300
+        img(src="img/blue-logo.svg" alt="" height="40")
+        a.upper.white.ml-40(href="{{rootURL}}") .info Explorer
+        a.upper.white.ml-40(href="https://www.blockchain.com/about") About
+        a.upper.white.ml-40(href="https://blog.blockchain.com") Blog
+        a.upper.white.ml-40(href="https://support.blockchain.com") Support
+      li.dropup(uib-dropdown)
```
Comment (jtormey) [inline app/partials/bc-logo-footer.pug]: This dropdown gets cut off by the header at smaller browser heights, could you fix that?

File: app/partials/bc-logo-header.pug (hunk 1 above)
Comment (plondon) [inline app/partials/bc-logo-header.pug]: Probably don't need it

File: app/partials/bc-logo-footer.pug (hunk 2 above)
Comment (plondon) [inline app/partials/bc-logo-footer.pug]: Fixed https://github.com/blockchain/brand-assets/commit/00282b20daafb506c9757953674052cd4dfbaf41

**Pull request merged**